Strategy11/formidable-forms
# Show save and reload modal on field type change

Fix https://github.com/Strategy11/formidable-pro/issues/5339
Related update: https://github.com/Strategy11/formidable-pro/pull/5443
### Test Procedure
1. Create a form and add a Product field and a different type field.
2. Go to the field options for those fields one by one and try changing the field type in Advanced > Field Type option.
3. Confirm that the 'Save and Reload' modal shows up when changing the field type and clicking on the each button in the modal does what is expected.

### Screen record
![CleanShot 2024-10-02 at 14 31 36](https://github.com/user-attachments/assets/7b9df585-9550-45b6-895c-93f5c9634601)
